Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -169,22 +171,33 @@ public static async Task<(int totalCount, int newCount)> ImportAccountAsync(Func
         private static async Task<List<ImportItem>> scanAccountsAsync(Func<Account, Task<ApiExtended>> apiExtendedfunc, Account[] accounts, LibraryOptions libraryOptions)
         {
             var tasks = new List<Task<List<ImportItem>>>();
-            foreach (var account in accounts)
+
+            LogArchiver archiver = default;
+
+            if (Log.IsEnabled(Serilog.Events.LogEventLevel.Verbose))
```

Comment:
Personal preference: `Log.Logger.IsVerboseEnabled()` extension in Dinah.Core. You're choice. Just wanted to point it out in case you want it also.

Reply:
I changed it.